test/Bridges/Constraint/norm_spec_nuc_to_psd.jl

```diff
@@ -0,0 +1,133 @@
+using Test
+
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIT = MathOptInterface.Test
+const MOIU = MathOptInterface.Utilities
+const MOIB = MathOptInterface.Bridges
+
+include("../utilities.jl")
+
+mock = MOIU.MockOptimizer(MOIU.UniversalFallback(MOIU.Model{Float64}()))
+config = MOIT.TestConfig()
+
+@testset "NormSpectral" begin
+    bridged_mock = MOIB.Constraint.NormSpectral{Float64}(mock)
+
+    MOIT.basic_constraint_tests(bridged_mock, config,
+        include = [(F, MOI.NormSpectralCone) for F in [
+            MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}, MOI.VectorQuadraticFunction{Float64}
+        ]])
+
+    inv6 = inv(6)
+    rt3 = sqrt(3)
+    invrt12 = inv(2 * rt3)
+    psd_dual = [inv6, -invrt12, inv6, invrt12, -invrt12, inv6, 0, 0, 0, 0, -invrt12, invrt12, -invrt12, 0, 0.5]
+    mock.optimize! = (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [rt3],
+        (MOI.VectorAffineFunction{Float64}, MOI.PositiveSemidefiniteConeTriangle) => [psd_dual])
+
+    MOIT.normspec1test(bridged_mock, config)
```

Comment:
@blegat are both of the norm spec tests supposed to be run here, or only the one that is used for the mock/bridged models below?

Reply:
On test is enough if you feel that it provides enough coverage

Reply:
thanks